src/main/java/com/selfxdsd/storage/SelfContracts.java

```diff
@@ -289,4 +300,28 @@ private Contract buildContract(final Record rec){
             rec.getValue(SLF_CONTRACTS_XDSD.ROLE),
             this.storage);
     }
+
+    /**
+     * Transaction Result compatible.
+     * Transactions apparently are not working well with jooq MySQL dialect on
+     * testing H2.
+     * @param jooq DSLContext.
+     * @param callable TransactionalCallable.
+     * @param <R> Result Type.
+     * @return Result.
+     * @throws Throwable If something went wrong and rollback is triggered.
+     * @checkstyle IllegalThrows (10 lines).
+     */
+    private <R> R transactionResultCompat(
```

Comment:
@criske If this method is really needed, I would name it simply ``transaction(...)``.

Reply:
@amihaiemil Is just a terminlogy from android (making newer features be compatible with older apis or other devices). just like polyfills shims from js.